lib/commands/debug.ts

```diff
@@ -1,14 +1,47 @@
 ﻿export class DebugPlatformCommand implements ICommand {
 	constructor(private debugService: IDebugService,
 		private $devicesService: Mobile.IDevicesService,
-		private $usbLiveSyncService: ILiveSyncService,
+		private $injector: IInjector,
 		private $logger: ILogger,
+		private $childProcess: IChildProcess,
+		private $devicePlatformsConstants: Mobile.IDevicePlatformsConstants,
 		protected $options: IOptions) { }
 
 	execute(args: string[]): IFuture<void> {
-		if (!this.$options.rebuild) {
+		if (!this.$options.rebuild && !this.$options.start) {
 			this.$options.debug = true;
-			return this.$usbLiveSyncService.liveSync(this.$devicesService.platform);
+			let usbLiveSyncService: ILiveSyncService = this.$injector.resolve("usbLiveSyncService");
+			let liveSyncServiceBase: any = this.$injector.resolve("liveSyncServiceBase");
+			let liveSyncProvider: ILiveSyncProvider = this.$injector.resolve("liveSyncProvider");
+
+			liveSyncServiceBase.on("sync", (device: Mobile.IDevice, data: ILiveSyncData) => {
+				let platformLiveSyncService: IPlatformLiveSyncService = this.$injector.resolve(liveSyncProvider.platformSpecificLiveSyncServices[data.platform.toLowerCase()], { _device: device });
+				let projectData: IProjectData = this.$injector.resolve("projectData");
+				let appId = device.isEmulator ? projectData.projectName : data.appIdentifier;
+				if (data.platform === this.$devicePlatformsConstants.iOS) {
+					try {
+						this.$childProcess.exec("/usr/bin/killall 'NativeScript Inspector'").wait();
+					} catch (e) {
```

Comment:
you can add:
```TypeScript
this.$logger.trace(`Error while killing NativeScript inspector: ${e.message}`);
```
So in case there's some strange error on client side, we'll be able to at least see it when `--log trace` is passed.